apiary.apib

```diff
@@ -242,6 +242,14 @@ The response to this request is a well-formed HTML document used as content for
 Use this URL as the `src` attribute value on an `<iframe>` element to embed an Unmade-powered product in an
 external e-commerce website.  Note that there are optional query parameters.
 
+|Query Parameter|Required?|Description|
+|---|---|---|
+|`price`| *optional* | The price to display for the product |
+|`ccy`| *optional* | The ISO 4217 currency code corresponding to the `price` |
+|`locale`| *optional* | The IETF language tag to set localisation of the user interface. Default: `en-GB`. |
+|`design_id`| *optional* | An Unmade Design ID that is compatible with this editor to be used as starting point |
+|`price`| *optional* | A JWT payload encrypted with your shared key which includes a unique identifier for your customer. See [advanced setup](https://unmade.docs.apiary.io/#introduction/advanced-setup) for more information. |
```

Comment:
Looks like first column here needs to be `init` not `price`.

`init` is also just the default key for this purpose, it's configurable per-embed.

Reply:
Good spot - fixed.

Reply:
@BenVosper this question is probably beyond the scope of this PR but what is the benefit of `init` being configurable? Is there a good reason for us to continue to want it to be so? I ask because it is clearly more complex to explain that a query parameter can be configured to have a different key.

Reply:
[Here](https://github.com/unmadeworks/embed-service/commit/f6ad327c247408796a6f75cfeff13613f3010373) is where that configurability was added. I can't see any explanation for why it was decided to do it that way.

If we can't find any examples where we're not using the default, we could probably remove it

Reply:
PR issued to remove this option https://github.com/unmadeworks/embed-service/pull/2728